Windows/GEDebugger/TabState.cpp

```diff
@@ -727,16 +727,16 @@ void FormatStateRow(wchar_t *dest, const TabStateRow &info, u32 value, bool enab
 
 	case CMD_FMT_TEXLEVEL:
 		{
-			const char *mipLevelModes[] = {
-				"auto + bias",
-				"bias",
-				"slope + bias",
-			};
 			const int mipLevel = value & 0xFFFF;
 			const int biasFixed = (s8)(value >> 16);
 			const float bias = (float)biasFixed / 16.0f;
 
 			if (mipLevel == 0 || mipLevel == 1 || mipLevel == 2) {
+				const char* mipLevelModes[] = {
```

Comment:
same, and more in this file

Reply:
Also it's not really needed to move those into the closest scope, since they'll be preinitialized anyway.

Reply:
fix it here https://github.com/hrydgard/ppsspp/pull/18581/commits/95f535dab0c7513af191a8b472bd1ab2ed58e271